packages/sdks/src/functions/is-from-trusted-host.ts

```diff
@@ -10,6 +10,9 @@ export function isFromTrustedHost(
   trustedHosts: string[] | undefined,
   e: { origin: string }
 ): boolean {
+  if (!e.origin.includes('http') && !e.origin.includes('https')) {
```

Comment:
why not `startsWith`? and can origin be `null` or `undefined`?

Reply:
thanks! let me update to `startsWith`

![Screenshot 2024-07-01 at 8 48 34 PM](https://github.com/BuilderIO/builder/assets/73601258/48a37935-ec15-4ce8-abe1-6177e909b4fd)
must be a string (https://developer.mozilla.org/en-US/docs/Web/API/MessageEvent/origin)